app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    list_of_command = command.split()
+    if list_of_command[1] != list_of_command[2] and list_of_command[0] == "cp":
```

Comment:
Validate command length

Then use unpacking to avoid accessing command parts using indexes